docs/source/upgrade_notes.rst

```diff
@@ -3,6 +3,15 @@
 Upgrade Notes
 =============
 
+.. _ref-upgrade-notes-v3-6:
+
+|st2| v3.6
+----------
+
+* Action delete API has been modified to delete related files from disk along with
+  de-registering actions from database. For action delete CLI command two arguments
```

Comment:
@mahesh-orch I'd maybe rephrase slightly on the CLI command. I'd mention that the action delete CLI will now wait for approval to delete on disk before proceeding, unless you use one of the -f or --force commands. As we want to get across the difference to those already using that command. 
If they are using it in a script they need to know that they would need to update to use the one of the new parameters, otherwise their existing scripts would need updating.